### PR TITLE
OSDOCS-8014: Admin Network Policy as Tech Preview release note

### PR DESCRIPTION
**Pull request by ahardin-rh**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-8014

Links to docs previews:

Feature description - https://65846--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-admin-network-policy

Tech Preview table update - https://65846--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-technology-preview

QE already approved this language in the sister PR https://github.com/openshift/openshift-docs/pull/65682

### CODE REVIEW

**Review by tssurya — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -304,10 +304,17 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
...
+Support for the Broadcom BCM57504 network interface controller is now available for the SR-IOV Network Operator.
 
 For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
 
+[id="ocp-4-14-admin-network-policy"]
+==== Admin Network Policy (Technology Preview)
+
+Admin Network Policy is available as a Technology Preview feature.
+
+You can enable `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy V2 API, in clusters running the OVN-Kubernetes CNI plugin. Cluster administrators can apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary. Currently, these APIs support only expressing policies for intra-cluster traffic.
```
Comment (tmalove) [inline release_notes/ocp-4-14-release-notes.adoc:316]:
```suggestion
You can enable `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy V2 API, in clusters running the OVN-Kubernetes CNI plug-in. Cluster administrators can apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary. Currently, these APIs support only expressing policies for intra-cluster traffic.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#p

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (ahardin-rh) [reply release_notes/ocp-4-14-release-notes.adoc:316]: @tmalove In this case, the IBM style guide is superseded by Red Hat style: https://redhat-documentation.github.io/supplementary-style-guide/#plugin

**Pull request merged**